DayPicker/tests/DayPicker-specs.js

```diff
@@ -1,47 +1,40 @@
 import ilib from '@enact/i18n';
-import {mount} from 'enzyme';
+import {fireEvent, render} from '@testing-library/react';
 
 import DayPicker, {getSelectedDayString} from '../DayPicker';
 
-
 describe('DayPicker', () => {
 
 	test('should not select any item when there is no passed prop `selected`', () => {
-		const subject = mount(
-			<DayPicker />
-		);
-		const selected = subject.find('CheckboxItem').find({'selected': true});
-		expect(selected.debug()).toBeFalsy();
+		const {getAllByRole} = render(<DayPicker />);
+
+		expect(getAllByRole('checkbox', {checked: false}));
 	});
 
 	test('should select day when passed prop `selected`', () => {
-		const subject = mount(
-			<DayPicker selected={[1]} />
-		);
-		const secondCheckboxItem = subject.find('CheckboxItem').find({'data-index': 1}).first();
-		const selected = secondCheckboxItem.props().selected;
-		expect(selected).toBe(true);
+		const {getAllByRole} = render(<DayPicker selected={[1]} />);
+		const selectedDay = getAllByRole('checkbox')[2];
```

Comment:
I saw why it should be 2 instead of 1.
I found it is because there is child checkbox in item.
It would be better is there is another selector using data-index instead of "getAllByRole" 
But I have not found it. :(